.circleci/config.yml

```diff
@@ -49,7 +49,8 @@ jobs:
             . venv/bin/activate
             pip install --install-option="--no-cython-compile" cython
             pip install numpy
-            pip install nose mpmath argparse Pillow codecov matplotlib "Sphinx!=3.1.0" asv pythran
+            pip install -r doc_requirements.txt
+            pip install nose mpmath argparse Pillow codecov asv pythran
```

Comment:
Both `nose` and `argparse` are not needed. Less sure about `Pillow`, but I'll check - we don't import it anywhere.